code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -1158,7 +1166,8 @@
 
 /datum/reagent/mental/neurapan/overdose(var/mob/living/carbon/M, var/alien)
 	M.add_chemical_effect(CE_PACIFIED, 1)
-	M.eye_blurry = max(M.eye_blurry, 30)
+	if(!locate(/datum/reagent/oculine) in M.reagents.reagent_list)
```

Comment:
I think these things should be chem effects rather than direct typechecks for a certain drug.